phpmyfaq/admin/section.php

```diff
@@ -62,31 +66,32 @@
               <?= $PMF_LANG['ad_menu_section_administration'] ?>
           </h1>
           <div class="btn-toolbar mb-2 mb-md-0">
-            <div class="btn-group mr-2">
-              <a class="btn btn-sm btn-outline-success" href="?action=group&amp;group_action=add">
-                  <?= $PMF_LANG['ad_group_add_link'] ?>
+            <div class="btn-section mr-2">
```

Comment:
I think the CSS class is still "btn-group"

Reply:
done